cli_reference/index.adoc

```diff
@@ -0,0 +1,39 @@
+[id="cli-tools-overview"]
+= {product-title} CLI tools overview
+include::modules/common-attributes.adoc[]
+:context: cli-tools-overview
+
+toc::[]
+
+A user performs different tasks while working on {product-title} which includes a range of operations, for example:
+
+* Managing clusters
+* Building, deploying and managing applications
+* Managing deployment processes
+* Developing operators
+* Creating and maintaining catalogs
```

Comment:
I might get more specific here by saying "Operator catalogs", if it's referring to `opm`.

Reply:
Yes it does. changed.